### PR TITLE
<FloatingNotification/> - migration to stylable

### PR DESCRIPTION
**Pull request by egidijusja**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6494

File: src/FloatingNotification/FloatingNotification.st.css [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+:global([dir='rtl']) .text,
+:global([dir='rtl']) .textButton {
+  margin-left: unset;
+  margin-right: 6px;
+}
+
+:global([dir='rtl']) .button {
+  margin-left: unset;
+  margin-right: 12px;
+}
```
Comment (EgoziE) [inline src/FloatingNotification/FloatingNotification.st.css:130]: I see all our :global(.rtl) styles here are related to margin and padding.
Let's remove all of them in favor of:

- margin-block-start
- margin-block-end
- padding-block-start
- padding-block-end

Note, In order to follow our browser support guidelines, don't use `margin-block` or `padding-block` without the starts/end suffix

Comment (EgoziE) [reply src/FloatingNotification/FloatingNotification.st.css:130]: You can take this PR as reference if needed - https://github.com/wix/wix-style-react/pull/6432

**Review by EgoziE — CHANGES_REQUESTED** [review]

File: src/FloatingNotification/FloatingNotification.st.css (hunk 1 above)
Comment (egidijusja) [reply src/FloatingNotification/FloatingNotification.st.css]: Updated with margin-inline-start, margin-inline-end, etc. to be able to remove :global(.rtl) styles.

File: src/FloatingNotification/FloatingNotification.st.css [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+  border-right: none;
+  max-width: 100%;
+  width: 100%;
+  height: 42px;
+  padding-inline-start: 30px;
+  padding-inline-end: 15px;
+}
+
+.root:type(standard) {
+    background-color: value(B50);
```
Comment (mykas) [inline src/FloatingNotification/FloatingNotification.st.css:37]: Lets fix those empty spaces

File: src/FloatingNotification/FloatingNotification.st.css [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    border-color: value(B30);
+}
+
+.root:type(success) {
+    background-color: value(G50);
```
Comment (mykas) [inline src/FloatingNotification/FloatingNotification.st.css:42]: Lets fix those empty spaces

**Review by mykas — APPROVED** [review]
Awesome job!

**Pull request merged**